stories/atoms/Icon/Icon.stories.jsx

```diff
@@ -103,57 +107,54 @@ Icon.args = {
     type: 'bc-icon-sound'
 };
 
+const IconCard = ({ name, copyContent }) => {
```

Comment:
This is a different component, you can write it in another file.

Reply:
For now, it's best to keep the current file structure unchanged. We may reconsider this approach in the future if the need arises.